generic_scripts/nlk.py

```diff
@@ -0,0 +1,167 @@
+import numpy as _np
+import matplotlib.pyplot as _plt
+from mathphys.constants import vacuum_permeability
+from mathphys.beam_optics import beam_rigidity
+
+
+class Wire:
+    """."""
+    def __init__(self, position, current):
```

Comment:
include docsctrings in all methods to explain them and the type and format of the inputs.

Reply:
Done!